request/en.txt

```diff
@@ -0,0 +1,65 @@
+Dear ○○,
+
+This is a staff of a Ruby 25th anniversary event.
+Ruby was born in 24th Feb, 1993, so will be 25th birthday.
+We held a event that celebrating and talking about Ruby's past, now and future, on 24th Feb. 2018, in Tokyo.
+
+We're also make a event on web collecting messages for Ruby from over the world Rubyist.We'd like to ask you to write message for Ruby and Rubyist, though I know how busy you are. We are planning to open the messages web pages on 1st Feb.
+
+## About the message
+
+About Ruby
+We don't make constraint for detail
+It's good to write past, now and future.
+License: CC BY-SA
+https://creativecommons.org/licenses/by-sa/2.0/
+
+And we make code of conduct in the event, so the message is also under the that.
+*** TODO: CoC url
+
+## Format
+
+There're 2 ways.
+
+### 1. GitHub Flavored Markdown
+
+GitHub Flavored Markdown: https://guides.github.com/features/mastering-markdown/
+Please place big files like an image or movie outside of repository, to prevent repository size will be huge.
+file name: id_or_name.language.md (example: igaiga.en.md)
+file format: github markdown
+
+There is a template file, but you can write freely without the template.
+https://raw.githubusercontent.com/ruby-no-kai/ruby25/master/docs/template.en.md
+
+### 2. Write in your blog page
+
+You can write in your blog page.
+Send us your file with your URL.
+File name and format rules are same as "1. GitHub Flavored Markdown".
+
+## Submission
+
+There're 2 ways.
+
+### a. Pull Request
+
+https://github.com/ruby-no-kai/ruby25
+Make a pull request to put your file named "id_or_name.language.md" into root directory in the "ruby-no-kai/ruby25" Github repository.
```

Comment:
Make a pull request and put your submission file into root directory in the "ruby-no-kai/ruby25" Github repository.  Your file name should be in the format "id_or_name.language.md".